### ISSUE TITLE
"Uninstall pack" when uninstalling extension

### ISSUE THREAD

**Issue by chrmarti** [Microsoft/vscode#55040]
Issue Type: <b>Bug</b>

Testing #54862

When uninstalling Azure Functions and then choosing to uninstall the pack, did only uninstall the extension:
![image](https://user-images.githubusercontent.com/9205389/43197227-d871df2e-900a-11e8-99ae-30fd85aae2b8.png)

VS Code version: Code - Insiders 1.26.0-insider (5fae91d61f12726e149fe5baed342eac2fbf8f88, 2018-07-25T01:56:56.548Z)
OS version: Darwin x64 17.7.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-4770HQ CPU @ 2.20GHz (8 x 2200)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: enabled<br>rasterization: enabled<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|3, 4, 4|
|Memory (System)|16.00GB (0.20GB free)|
|Process Argv|/Applications/Visual Studio Code - Insiders.app/Contents/MacOS/Electron|
|Screen Reader|no|
|VM|22%|

</details><details><summary>Extensions (24)</summary>

Extension|Author (truncated)|Version
---|---|---
autofilename|Jer|0.1.3
999-pack1|chr|0.12.2
regex|chr|0.2.0
ssh|chr|0.0.4
vscode-eslint|dba|1.4.12
Spot|der|0.4.0
ghlink|dt|1.0.2
EditorConfig|Edi|0.12.4
tslint|eg2|1.0.34
rest-client|hum|0.19.0
javascript-unit-test-snippet|iZD|1.0.0
vscode-kitchen|jir|0.5.0
Angular2Tests|Mar|0.7.3
vscode-cosmosdb|ms-|0.8.0
azure-account|ms-|0.4.2
azurecli|ms-|0.4.2
github-issues-prs|ms-|0.9.0
PowerShell|ms-|1.8.1
debugger-for-chrome|msj|4.8.2
vscode-docker|Pet|0.0.27
vscode-versionlens|pfl|0.21.1
999-extension-pack1|san|0.0.2
todotasks|san|0.5.0
vscode-open-in-github|sys|1.7.0

(1 theme extensions excluded)

</details>

**Comment by sandy081** [comment]
Because an another extension `Azure Cosmons DB` is depending on `Azure Account` extension and hence it is excluded from uninstalling.

**Comment by chrmarti** [comment]
`Azure Cosmos DB` is also in the pack. I would have expected that to be uninstalled too. (Reopening for discussion.)
```json
    "extensionPack": [
        "sandy081.999-extension-pack1",
        "JerryHong.autofilename",
        "chrmarti.regex",
        "chrmarti.ssh",
        "dbaeumer.vscode-eslint",
        "derek-bekoe.Spot",
        "dt.ghlink",
        "EditorConfig.EditorConfig",
        "eg2.tslint",
        "humao.rest-client",
        "ms-azuretools.vscode-azureappservice",
        "ms-azuretools.vscode-azurefunctions",
        "ms-azuretools.vscode-cosmosdb",
        "ms-vscode.azure-account",
        "ms-vscode.azurecli",
        "ms-vscode.github-issues-prs",
        "ms-vscode.PowerShell",
        "msjsdiag.debugger-for-chrome",
        "PeterJausovec.vscode-docker",
        "pflannery.vscode-versionlens",
        "sysoev.vscode-open-in-github",
        "vsciot-vscode.azure-iot-toolkit"
    ]
```

**Comment by sandy081** [comment]
This is fixed by #55055. In this case, there will be no prompt shown while uninstalling `Azure Functions` and uninstall only this as `Azure Cosmos DB`  extension depends on `Azure Account` extension.

@chrmarti Please try with the fix and check
